travis/push-to-pypi.sh

```diff
@@ -35,7 +35,7 @@ if [ -n "$PYPI_RELEASE" ] ; then
     #twine upload --repository-url https://pypi.org/legacy/ -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
     python setup.py sdist
     twine upload -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
-    $SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}"
+    $SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}" "true"
```

Comment:
you will also need to make changes when push-to-cicd-status.sh is called from build-push-aci-containers-image.sh , acc-provision-operator-image.sh, build-push-opflex-images.sh